SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift

```diff
@@ -0,0 +1,424 @@
+    commonSelector.action = #selector(onSegmentSelected)
+  }
+
+  func numberOfRows(in tableView: NSTableView) -> Int {
+    if(tableView == defaultTable) {
+      return 3
+    }
+    else if(tableView == commonTable) {
+      return 5
+    }
+    return 4
+  }
+
+  func tableView(_ tableView: NSTableView, viewFor tableColumn: NSTableColumn?, row: Int) -> NSView? {
+    tableView.headerView=nil
+
+    // Common schema properties section
+    if(tableView.tag == Section.CommonSchemaProperties.rawValue) {
+      if let cell = tableView.make(withIdentifier: "property", owner: nil) as? NSTableCellView {
+        let property = propertyAtRow(row: row)
+        switch (row) {
+        case kDeviceIdRow:
+          let key: NSTextField = cell.subviews[0] as! NSTextField
+          key.stringValue=property.key
+          let value: NSButton = cell.subviews[1] as! NSButton
+          let selectedTarget = selectedTransmissionTarget(commonSelector)
+          value.state = (collectDeviceIdStates[selectedTarget!]?.hashValue)!
+          value.isEnabled = !((value.state as NSNumber).boolValue)
+          value.action = #selector(collectDeviceIdSwitchCellEnabled)
+          cell.subviews[2].isHidden = true
+          return cell   
+        case kAppNameRow:
+          let key: NSTextField = cell.subviews[0] as! NSTextField
+          key.stringValue = property.key
+          let value: NSTextField = cell.subviews[2] as! NSTextField
+          value.stringValue = property.value
+          cell.subviews[1].isHidden = true
+          return cell
+        case kAppVersionRow:
+          let key: NSTextField = cell.subviews[0] as! NSTextField
+          key.stringValue = property.key
+          let value: NSTextField = cell.subviews[2] as! NSTextField
+          value.stringValue = property.value
+          cell.subviews[1].isHidden = true
+          return cell
+        case kAppLocaleRow:
+          let key: NSTextField = cell.subviews[0] as! NSTextField
+          key.stringValue = property.key
+          let value: NSTextField = cell.subviews[2] as! NSTextField
+          value.stringValue = property.value
+          cell.subviews[1].isHidden = true
+          return cell
+        case kUserIdRow:
+          let key: NSTextField = cell.subviews[0] as! NSTextField
+          key.stringValue = property.key
+          let value: NSTextField = cell.subviews[2] as! NSTextField
+          value.stringValue = property.value
+          cell.subviews[1].isHidden = true
+          return cell
+        default:
+           return nil
+        }
+      }
+      return nil
+    }
+
+    //Transmission target section
+    if let cell = tableView.make(withIdentifier: "target", owner: nil) as? NSTableCellView {
+      let section = transmissionTargetSections![tableView.tag]
+      switch (row) {
+      case kEnabledCellRowIndex:
+        let key: NSTextField = cell.subviews[0] as! NSTextField
```

Comment:
The cell subview indices should also be variables

Reply:
Add enum `cellSubviews` for subview index.